### ISSUE TITLE
[terra-form-select] Invalid attribute value for aria-activedescendant

### ISSUE THREAD

**Issue by vb057454** [cerner/terra-core#2765]
# Issue Description
Verified using axe on the terra core site, saw the same error just like the below stacktrace. Also, attached screenshots to better describe the issue.
expected no accessibility violations but got:
	{
  "description": "Ensures all ARIA attributes have valid values",
  "help": "ARIA attributes must conform to valid values",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/aria-valid-attr-value?application=axeAPI",
  "id": "aria-valid-attr-value",
  "impact": "critical",
  "nodes": [
    {
      "all": [
        {
          "data": [
            "aria-activedescendant=\"terra-select-option-null\""
          ],
          "id": "aria-valid-attr-value",
          "impact": "critical",
          "message": "Invalid ARIA attribute value: aria-activedescendant=\"terra-select-option-null\"",
          "relatedNodes": []
        }
      ],
      "any": [],
      "failureSummary": "Fix all of the following:\n  Invalid ARIA attribute value: aria-activedescendant=\"terra-select-option-null\"",
      "html": "<ul role=\"listbox\" class=\"_Menu-module__menu___2E516\" aria-activedescendant=\"terra-select-option-null\" tabindex=\"0\"><div class=\"_NoResults-module__no-results___cl0WV\"><div>health-benefits-management-engine.network-error</div></div></ul>",
      "impact": "critical",
      "none": [],
      "target": [
        "ul"
      ]
    }
  ],
  "tags": [
    "cat.aria",
    "wcag2a",
    "wcag412"
  ]
}

## Additional Context / Screenshots
![image](https://user-images.githubusercontent.com/46331155/69755394-bc5a9c00-111d-11ea-9ea5-9e054b410e9b.png)

**Comment by yuderekyu** [comment]
What version of axe-core does your project use?

**Comment by vb057454** [comment]
3.2.2

**Comment by saket2403** [comment]
I  was unable to re-create this issue on our site using a newer version of axe. In fact, toolkit is being upgraded to use axe version `3.4.0` and above. Here is the [issue link](https://github.com/cerner/terra-toolkit/issues/343) for you to keep track.This will be out soon once we fix the issues due to new rules in `v3.4.0`.
PS: If the wdios are failing in your project due to this rule you can disable it for now till toolkit is released.
cc @vb057454

**Comment by saket2403** [comment]
Can we close this issue @yuderekyu ?

**Comment by StephenEsser** [comment]
This appears to be an issue on both versions of axe-core.

Here is a branch to reproduce the violation running axe-core 3.2.0.

https://github.com/cerner/terra-core/tree/select-no-results-accessibility-control

Here is the branch upgraded to axe-core 3.4.0 that logs the same violation.

https://github.com/cerner/terra-core/tree/select-no-results-accessibility-axe-3.4.0

Briefly looking into this I think this issue can be resolved by checking for the active option before assigning the aria attribute here: https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/src/multiple/Menu.jsx#L254

**Comment by saket2403** [comment]
Thanks for the heads up @StephenEsser. I have a PR out to fix this.
